### PR TITLE
Devops2.0 helm chart

### PR DESCRIPTION
**Pull request by antonioT90**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  allowPrivilegeEscalation: false
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+
+service:
+  type: ClusterIP
+  port: 8088
```
Comment (diegolagospagopa) [inline helm/values.yaml:43]: Please in this case that we have 2 service use something like "appPort" and "appTargetPort"

File: helm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    - host: ""
+      paths:
+        - path: "/(party-management)/v1/(.*)"
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
```
Comment (diegolagospagopa) [inline helm/values.yaml:69]: Please enabled this attribute to be able to manage the load, and avoid problems with the pod
https://github.com/pagopa/template-microservizio-k8s/blob/e7a198f1125ac53b7fc61154851de26dac4890fa/helm/values-prod.yaml#L26

File: .devops/deploy-pipelines.yml [hunk 3]
```diff
@@ -85,15 +106,15 @@ stages:
               command: 'Build'
               Dockerfile: '$(Build.Repository.LocalPath)/$(dockerfileRelativePath)'
               buildContext: '$(Build.Repository.LocalPath)'
-              arguments: --build-arg APP_IMAGE=$(DOCKER_BASE_IMAGE_NAME):${{ parameters.version }}
+              arguments: --build-arg APP_IMAGE=$(DOCKER_BASE_IMAGE_NAME):$(dockerImageVersion)
               tags: |
                 $(Build.BuildId)
```
Comment (diegolagospagopa) [inline .devops/deploy-pipelines.yml:111]: can we add the branch name to be more simple to understand the buildId?

File: .devops/deploy-pipelines.yml [hunk 4]
```diff
@@ -107,18 +128,14 @@ stages:
...
```
Comment (diegolagospagopa) [inline .devops/deploy-pipelines.yml:129]: Same as before for the name of the branch

File: .devops/deploy-pipelines.yml (hunk 3 above)
Comment (antonioT90) [reply .devops/deploy-pipelines.yml:111]: the branch name in this case will be always "main"

File: .devops/deploy-pipelines.yml (hunk 4 above)
Comment (antonioT90) [reply .devops/deploy-pipelines.yml:129]: the branch name in this case will be always "main"

File: helm/values.yaml (hunk 2 above)
Comment (antonioT90) [reply helm/values.yaml:69]: in order to provide a limit on the resources we should talk to every responsible of BE application to ask for suggestions

File: .devops/deploy-pipelines.yml (hunk 3 above)
Comment (antonioT90) [reply .devops/deploy-pipelines.yml:111]: we could user buildNumber? (eg: 20220310.14, or the date plus the number of execution at the date)

**Review by diegolagospagopa — CHANGES_REQUESTED** [review]

File: .devops/deploy-pipelines.yml (hunk 3 above)
Comment (antonioT90) [reply .devops/deploy-pipelines.yml:111]: done

File: .devops/deploy-pipelines.yml [hunk 5]
```diff
@@ -133,35 +150,29 @@ stages:
...
+                    chartPath: $(Pipeline.Workspace)/helm
+                    chartName: $(K8S_IMAGE_REPOSITORY_NAME)
+                    releaseName: $(K8S_IMAGE_REPOSITORY_NAME)
+                    valueFile: "$(Pipeline.Workspace)/helm/values-${{ lower(variables.environment) }}.yaml"
+                    install: true
+                    waitForExecution: true
+                    arguments: "--timeout 2m0s"
                 - task: KubernetesManifest@0
-                  displayName: Deploy to Kubernetes cluster
+                  displayName: Patch
```
[the remark this answers is not in the document]
Comment (antonioT90) [reply .devops/deploy-pipelines.yml:170]: done

File: helm/values.yaml [hunk 6]
```diff
@@ -0,0 +1,97 @@
+# Default values for selfcare-uservice-party-management.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: selc
+
+replicaCount: 1
```
[the remark this answers is not in the document]
Comment (antonioT90) [reply helm/values.yaml]: moved inside values-ENV.yaml

File: .devops/deploy-pipelines.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (antonioT90) [reply .devops/deploy-pipelines.yml]: displayName updated

**Review by diegolagospagopa — APPROVED** [review]

**Review by diegolagospagopa — APPROVED** [review]

**Pull request merged**